app/customer.py

```diff
@@ -0,0 +1,37 @@
+from __future__ import annotations
+import dataclasses
+from typing import List
+from app.data_jason import load_json
+
+
+@dataclasses.dataclass
+class Person:
+    name: str
+    location: List[int]
+    money: int | float
+    car_brand: str
+    fuel_consumption: float | int
+    product: dict
+    fuel_price: int | float
+
+    @classmethod
+    def load_person(cls) -> List[Person]:
```

Comment:
```suggestion
    def load_people(cls) -> List[Person]:
```

Reply:
`load_people` is a better name 
because you are load not one person but a list of them